Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -115,4 +136,48 @@ void HTMLLinkElement::resource_did_load()
     document().style_sheets().add_sheet(sheet.release_nonnull());
 }
 
+void HTMLLinkElement::resource_did_load_favicon()
+{
+    VERIFY(m_relationship & (Relationship::Icon));
+    if (!resource()->has_encoded_data()) {
+        dbgln_if(SPAM_DEBUG, "Favicon downloaded, no encoded data");
+        return;
+    }
+
+    dbgln_if(SPAM_DEBUG, "Favicon downloaded, {} bytes from {}", resource()->encoded_data().size(), resource()->url());
+
+    RefPtr<Gfx::Bitmap> favicon_bitmap;
+    auto decoded_image = Web::image_decoder_client().decode_image(resource()->encoded_data());
```

Comment:
Are you not supposed to do something with `decoded_image` here?

Reply:
I think it is no longer needed here, will remove